docs/src/tutorials/magnetism.ipynb

```diff
@@ -0,0 +1,577 @@
+    "These two layers are charachterized by having a scattering length density (SLD) of respectively 4 and 8.\n",
+    "Both layers have a rougness of 2 but their thicknesses are 100 and 150 angstrom respectively.\n",
+    "We show the model that will be used graphically below. \n",
+    "\n",
+    "<center>\n",
+    "    <img src='sample/two_layers.png' alt='A slab model description of the two_layers system.' width='300px'></img>\n",
+    "</center>\n",
+    "<center>\n",
+    "    A slab model description of the two layer.\n",
+    "</center>\n",
+    "\n",
+    "To construct such a layer structure, first we create each of the materials, the associated layers, and the sub and super phases. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0f95d620-35b7-4b47-a3b4-9e33d5525b50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sld_4 = Material(sld=4.0, isld=0, name='Sld 4')\n",
+    "sld_4_layer = Layer(material=sld_4, thickness=100, roughness=2, name='SLD 4 Layer')\n",
+    "\n",
+    "sld_8 = Material(sld=8.0, isld=0, name='Sld 8')\n",
+    "sld_8_layer = Layer(material=sld_8, thickness=150, roughness=2, name='SLD 8 Layer')\n",
+    "\n",
+    "vacuum = Material(sld=0, isld=0, name='Vacuum')\n",
+    "superphase = Layer(material=vacuum, thickness=0, roughness=2, name='Vacuum Superphase')\n",
+    "\n",
+    "si = Material(sld=2.047, isld=0, name='Si')\n",
+    "subphase = Layer(material=si, thickness=0, roughness=2, name='Si Subphase')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f63ec440-089f-46cf-8ff5-be5012ad8dc8",
+   "metadata": {},
+   "source": [
+    "We then create a model for the two layered structures."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2af8c30b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "two_layers = Multilayer([sld_4_layer, sld_8_layer], name='SLD 4/8 Layer')\n",
+    "sample = Sample(superphase, two_layers, subphase, name='Two Layer Sample')\n",
+    "model = Model(\n",
+    "    sample=sample,\n",
+    "    scale=1,\n",
+    "    background=0,\n",
+    "    name='Two Layer Model',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1d3ad93",
+   "metadata": {},
+   "source": [
+    "We also need a Refl1d sample"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0259cd0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "refl1d_sld_4 = refl1d.names.SLD(name=\"Sld 4\", rho=4.0, irho=0)\n",
+    "refl1d_sld_8 = refl1d.names.SLD(name=\"Sld 8\", rho=8.0, irho=0)\n",
+    "refl1d_vacuum = refl1d.names.SLD(name=\"Vacuum\", rho=0, irho=0)\n",
+    "refl1d_si = refl1d.names.SLD(name=\"Si\", rho=2.047, irho=0)\n",
+    "\n",
+    "# Refl1d model is inverted as compared to EasyReflectometry, so the order of the layers is reversed\n",
+    "refl1d_sample = (\n",
+    "    refl1d_si(0, 2) | \n",
```

Comment:
Same comment here about layer roughnesses, and making it clearer which one is not used

Reply:
changed roughness to 0